app/src/main/res/values-night/styles.xml

```diff
@@ -0,0 +1,37 @@
+<resources xmlns:tools="http://schemas.android.com/tools">
```

Comment:
what was the need for creating the night value styles.xml file if it was already working by using a colors.xml night value file?

Reply:
@rob729 the need was that the icons in status bar were not toggling on their own.
I had to set 
  "android:windowLightStatusBar" as falsefor dark mode and as true for light mode. Otherwise it wasn't functioning properly

Reply:
if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.M) {
                window.decorView.systemUiVisibility = View.SYSTEM_UI_FLAG_LIGHT_STATUS_BAR
            }
Using the above piece of code, you set the value of android:windowLightStatusBar programmatically.
Try to implement it in the main activity and tell if it works or not.

Reply:
okay

Reply:
I remember trying this method. It was giving "unresolved reference" error. And I didn't find any solid method to remove this error

<img src="https://user-images.githubusercontent.com/61552810/95083335-02b75900-073a-11eb-8749-aeca038a60fa.PNG"
width="700px" height="200px"/>

Reply:
I have mentioned that you have to add it to the main activity.

Reply:
Oops! now it's working fine. But one problem, it is making the icons dark, when the background is light. But not making them light when background is dark.

Reply:
so you will have to logically apply these lines of code.
I have not given the complete implementation to you.
Try to use it with the night mode sharedPref.

Reply:
@rob729 Ok, I'll try to figure it out. By the way why can't we use the style-night file creation method? Just wanted to know.

Reply:
Not an optimal approach if the same thing can be done using 3-4 lines of code.
It is just a repetitive code with very few changes.

Reply:
@rob729 I am able to set it for light mode or for dark mode separately but sorry I am not able to toggle them programmatically. I have also explained my problem in detail on the Gitter channel.